spiceaidocs/docs/components/data-connectors/clickhouse.md

```diff
@@ -4,36 +4,66 @@ sidebar_label: 'Clickhouse Data Connector'
 
 ## Configuration
 
+### `from`
+
+The `from` field for the Clickhouse connector takes the form of `from:db.dataset` where `db.dataset` is the path to the Dataset within Clickhouse. In the example above it would be `my.dataset`.
+
+If `db` is not specified in either the `from` field or the `clickhouse_db` parameter, it will default to the `default` database.
+
+### `name`
+
+The dataset name. This will be used as the table name within Spice.
+
+```yaml
+datasets:
+  - from: clickhouse:my.dataset
+    name: cool_dataset
+```
+
+```sql
+SELECT COUNT(*) FROM cool_dataset;
+```
+
+```shell
++----------+
+| count(*) |
++----------+
+| 6001215  |
++----------+
+```
+
+### `params`
+
 The Clickhouse data connector can be configured by providing the following `params`:
 
-- `clickhouse_connection_string`: The connection string to use to connect to the Clickhouse server. This can be used instead of providing individual connection parameters. Use the [secret replacement syntax](../secret-stores/index.md) to load the password from a secret store, e.g. `${secrets:my_clickhouse_conn_string}`.
-- `clickhouse_host`: The hostname of the Clickhouse server.
-- `clickhouse_tcp_port`: The port of the Clickhouse server.
-- `clickhouse_db`: The name of the database to connect to.
-- `clickhouse_user`: The username to connect with.
-- `clickhouse_pass`: The password to connect with. Use the [secret replacement syntax](../secret-stores/index.md) to load the password from a secret store, e.g. `${secrets:my_clickhouse_pass}`.
-- `clickhouse_secure`: Optional. Specifies the SSL/TLS behavior for the connection, supported values:
-  - `true`: (default) This mode requires an SSL connection. If a secure connection cannot be established, server will not connect.
-  - `false`: This mode will not attempt to use an SSL connection, even if the server supports it.
-- `connection_timeout`: Optional. Specifies the connection timeout in milliseconds.
+| Parameter Name                 | Definition                                                                                                                                                                                                                                                                                                              |
+| ------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `clickhouse_connection_string` | The connection string to use to connect to the Clickhouse server. This can be used instead of providing individual connection parameters.                                                                                                                                                                               |
```

Comment:
```suggestion
| `clickhouse_connection_string` | The connection string to use to connect to the ClickHouse server. This can be used instead of providing individual connection parameters.                                                                                                                                                                               |
```